metadata.yaml

```
@@ -1,50 +1,39 @@
 # Copyright 2025 Canonical Ltd.
 # See LICENSE file for licensing details.
-# This file populates the Overview on Charmhub.
-# See https://juju.is/docs/sdk/metadata-reference for a checklist and guidance.
-
-# The charm package name, no spaces (required)
-# See https://juju.is/docs/sdk/naming#heading--naming-charms for guidance.
-name: is-charms-template
- 
-# The following metadata are human-readable and will be published prominently on Charmhub.
-
-# (Recommended)
-display-name: Charm Template
-
-# (Required)
-summary: A very short one-line summary of the charm.
-docs: https://discourse.charmhub.io
-issues: https://github.com/canonical/is-charms-template-repo/issues
+name: hockeypuck-k8s
+assumes:
+  - juju >= 3.1
+  - k8s-api
+display-name: Hockeypuck K8s
+summary: Hockeypuck OpenPGP public keyserver
 maintainers: 
   - https://launchpad.net/~canonical-is-devops
-source: https://github.com/canonical/is-charms-template-repo
-
 description: |
-  A single sentence that says what the charm is, concisely and memorably.
-
-  A paragraph of one to three short sentences, that describe what the charm does.
-
-  A third paragraph that explains what need the charm meets.
-
-  Finally, a paragraph that describes whom the charm is useful for.
-
-# The containers and resources metadata apply to Kubernetes charms only.
-# Remove them if not required.
-
-# Your workload’s containers.
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+  for deploying and managing [Hockeypuck](https://hockeypuck.io/) on Kubernetes. Hockeypuck is an
+  OpenPGP public keyserver, a software tool used to manage public key infrastructure for PGP 
```

```suggestion
  OpenPGP public keyserver tool used to manage public key infrastructure for PGP 
```
Making an attempt for conciseness

---

done